modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/model/Contract.java

```diff
@@ -45,15 +45,15 @@
      * Representations available with this API Note: their "name" is used as a
      * reference further in this description.
      */
-    private List<Representation> representations;
+    private List<Representation> representations = new ArrayList<>();
```

Comment:
in swagger core, field is null by default, and add... method initializes it with an empty list if needed.
I think that pattern is good